src/test/java/th/ac/kmitl/science/comsci/example/models/TradeProductTest.java

```diff
@@ -0,0 +1,40 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class TradeProductTest {
+
+    TradeProduct tradeProduct = new TradeProduct();
+
+    @Test
+    public void testConstructor(){
+        String id = "id2780101";
+        String globalId = "GId2780101";
+        String name = "Laptop";
+        TradeProduct ConstructorTradeProduct = new TradeProduct(id,
+                                                                globalId,
+                                                                name);
+        assertEquals(id, ConstructorTradeProduct.getId());
+        assertEquals(globalId, ConstructorTradeProduct.getGlobalId());
+        assertEquals(name, ConstructorTradeProduct.getName());
+    }
+
+    @Test
+    public void testGetId() {
+        tradeProduct.setId("id2780101");
+        assertEquals("id2780101", tradeProduct.getId());
+    }
+
+    @Test
+    public void testGetGlobalId() {
+        tradeProduct.setGlobalId("GId2780101");
+        assertEquals("GId2780101", tradeProduct.getGlobalId());
+    }
+
+    @Test
+    public void testGetName() {
```

Comment:
These 3 test can be merge, why we need 3 test to just test getting and setting property ?